app/ui/src/main/res/values/strings.xml

```diff
@@ -786,6 +786,7 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="settings_message_theme_label">Message view theme</string>
     <string name="settings_compose_theme_label">Composer theme</string>
     <string name="settings_language_label">Language</string>
+    <string name="preference_search_no_results">No settings found</string>
```

Comment:
Is this overriding a string provided by the library? If so, please put it into a separate section, using a header similar to what we use here (include the library name): 
https://github.com/k9mail/k-9/blob/de1612fb16f51a074cfa12f5f12874cdc93d717c/app/ui/src/main/res/values/strings.xml#L66

Reply:
Interesting. I did not know that you can overwrite library strings. The string is not directly overwriting the library string. It uses `setTextNoResults(...)` of the library.

Reply:
My bad. I couldn't find a usage of the string. But it's clearly there :)